index.html

```diff
@@ -13,13 +13,17 @@
   <div class="wrapper">
     <!-- Header section starts here -->
     <section class="header-section">
+      <div class="background-blur-menu overlay-hide" id="overlay"></div>
       <header class="main-header">
         <div class="logo">
           <a href="#">Amar</a>
         </div>
         <div class="toggler" id="toggler"></div>
-        <nav class="nav">
-          <ul>
+        <nav class="nav nav-hide" id="show-nav">
+          <div class="nav-close" id="close">
+            <img src="./app/assets/pngs/icons/close.png" alt="" srcset="">
+          </div>
+          <ul class="menu">
             <li><a href="#">Portfolio</a></li>
             <li><a href="#">About</a></li>
             <li><a href="#">Contact</a></li>
```

Comment:
- [ ] Kindly make sure when any navigation link is clicked the menu is closed automatically and the user will be redirected to the part of the page according to the clicked nav link, if the `About` link is clicked the menu will be closed and the user must be redirected to the `About` section on your portfolio.

   ### YOUR WEBSITE BEHAVIOUR

    ![chrome-capture-2023-4-23 (2)](https://github.com/amarmahdi1112/amarmahdi1112.github.com/assets/87197412/29f3a107-aee5-42f0-869e-152f9305ebe5)